### ISSUE TITLE
Cannot find module 'currify/legacy'

### ISSUE THREAD

**Issue by vingorius** [coderaiser/cloudcmd#93]
* **Version** (`cloudcmd -v`): cloudcmd@5.11.1
* **Node Version** `node -v`: v4.3.0
* **OS** (`uname -a` on Linux): Ubuntu 16.04  kernel : 4.4.0-53-generic
* **Browser name/version**: NA

I installed Cloudcmd as Express middleware.
When start it, it faild, below is error log.

Thanks.

`
 ~/projects/cloud2 $ nodemon index.js
[nodemon] 1.11.0
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: *.*
[nodemon] starting `node index.js`
module.js:328
    throw err;
    ^

Error: Cannot find module 'currify/legacy'
    at Function.Module._resolveFilename (module.js:326:15)
    at Function.Module._load (module.js:277:25)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/vingorius/projects/cloud2/node_modules/cloudcmd/lib/server/plugins.js:3:15)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
[nodemon] app crashed - waiting for file changes before starting...

`

**Comment by coderaiser** [comment]
It is fixed in [currify v2.0.3](https://github.com/coderaiser/currify/releases/tag/v2.0.3). Could you please re-install `cloudcmd` with:

```sh
npm r cloudcmd && npm i cloudcmd
```

**Comment by vingorius** [comment]
Same Error, So i changed cloudcmd@5.10.1,finally it works.

**Comment by coderaiser** [comment]
Can not reproduce with fresh install of `v5.11.1`, you beter try to install `currify@2.0.3`.
Could you plese try global install. Is it work for you?

**Comment by vingorius** [comment]
I deleted all the node_modules, and clean install it again.

I attached install log file, as shown on log file, currify@2.0.3 was there.

But same error happened.

```
 ~/test/cloudcmdtest $ node index.js 
module.js:328
    throw err;
    ^

Error: Cannot find module 'currify/legacy'
    at Function.Module._resolveFilename (module.js:326:15)
    at Function.Module._load (module.js:277:25)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/vingorius/test/cloudcmdtest/node_modules/cloudcmd/lib/server/plugins.js:3:15)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
```

[install_log.txt](https://github.com/coderaiser/cloudcmd/files/656127/install_log.txt)

**Comment by coderaiser** [comment]
Looks like something is wrong with a cache. As you see `currify v2.0.3` installed 5 times for 5 modules. It is mean that for `cloudcmd` installed different version of `currify`.

Could you look at `node_modules/currify/package.json` what `version` it contains? And what version of `currify` mentioned in `dependencies` field of `node_modules/cloudcmd/packages.json`?

You can try to remove cache with:
```sh
npm cache clean
```
or
```
rm -rf ~/.npm
```
And install `Cloud Commander` with  `npm i cloudcmd --force`.

**Comment by coderaiser** [comment]
@vingorius is it help you?

**Comment by vingorius** [comment]
Hi @coderaiser ,There's no currify dependency field of node_modules/cloudcmd/packages.json?

**Comment by coderaiser** [comment]
Landed in [v5.12.1](https://github.com/coderaiser/cloudcmd/releases/tag/v5.12.1).

**Comment by vingorius** [comment]
@coderaiser OK, It works. thanks.
